cpu/saml21/cpu.c

```diff
@@ -277,10 +282,11 @@ void cpu_init(void)
 
     /* Setup GCLK generators */
 #if USE_DFLL
-    _gclk_setup(SAM0_GCLK_MAIN, GCLK_GENCTRL_GENEN | GCLK_GENCTRL_SRC_DFLL48M);
+#define GCLK_GENCTRL_SRC_MAIN GCLK_GENCTRL_SRC_DFLL48M
 #else
-    _gclk_setup(SAM0_GCLK_MAIN, GCLK_GENCTRL_GENEN | GCLK_GENCTRL_SRC_OSC16M);
+#define GCLK_GENCTRL_SRC_MAIN GCLK_GENCTRL_SRC_OSC16M
```

Comment:
Please move that to the top of the file where all the other defines live.

Reply:
Sure, done it.